SimPEG/EM/NSEM/SurveyNSEM.py

```diff
@@ -64,27 +83,31 @@ def eval(self, f):
     def evalDeriv(self, f):
         raise Exception('Use Sources to project fields deriv.')
 
-#################
+#########
 # Data
-#################
+#########
 
 
-class Data(SimPEGsurvey.Data):
-    '''
+class Data(SimPEGsurvey.Data, DataNSEM_plot_functions):
```

Comment:
@grosenkj: what are your thoughts on naming this `DataNSEM` and then importing it in the init for NSEM as `import DataNSEM as Data`. On further development, I think this might help with debugging as errors should be more transparent